### PR TITLE
Trim auth flows

### LINKED ISSUE #624: Incorrect routing on token retrieval
Occasionally (need solid repro), after refresh a submission will be sent to the upload page, no matter their target page.

Seems to be a problem with `restorePage`

### PR DESCRIPTION
**Pull request by wpears**
Closes #624 

Removes odd edge case behavior, unnecessary re-renders, and subtle visual bugs.

### CODE REVIEW

File: src/sass/components/Header.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.Header {
```
Comment (awolfe76) [inline src/sass/components/Header.scss:1]: Turning these off renders nearly (if not completely) identical ... or am I missing something?

Comment (wpears) [reply src/sass/components/Header.scss:1]: When refreshing the page the blue `Login` button initially is slightly wider than it's automatic width then shrinks. It's only on page load and lasts for <100ms but happens every time.

Comment (awolfe76) [reply src/sass/components/Header.scss:1]: Feels like an odd fix for such a small thing ... and I'm not seeing that happen at all.

Comment (wpears) [reply src/sass/components/Header.scss:1]: the margin and padding aren't actually required for the fix, but they help collapse the wrapper to the content, which I think it appropriate here.

The other change is just setting a defined width so the `auto` width doesn't recalculate during load.

**Pull request merged**